Sources/TensorFlow/Layer.swift

```diff
@@ -41,6 +43,31 @@ public extension Layer {
     }
 }
 
+public struct EmptyTangentVector: Differentiable & VectorProtocol & ElementaryFunctions & 
```

Comment:
Could you please add a doc comment to newly added types (`EmptyTangentVector` and `ParameterlessLayer`)?

You can use `X, Y` here instead of `X & Y` to save one character per composed protocol.
```suggestion
public struct EmptyTangentVector: Differentiable, VectorProtocol, ElementaryFunctions,
```

Reply:
Thanks Dan!